src/components/payment/buy-form.tsx

```diff
@@ -45,7 +45,26 @@ const _cardSchema = z.object({
   }),
 });
 
-const schema = _addressSchema.and(_cardSchema).and(dateSchema);
+export const aditionalAddressSchema = z.object({
+  adtionalCity: z.string({
```

Comment:
Beware of the word `additional`, it is missing a `d`.
```suggestion
  additionalCity: z.string({
```